test/docker-compose.yml

```diff
@@ -46,9 +46,20 @@ services:
       # For initial populating DB
       - GENERATE_TEST_DATA=true
 
-  # ui:
-  #   image: node
-  #   container_name: User-Interface
+  ui:
+    image: express-cuts-ui:latest
+    build:
+      context: ../frontend
+    container_name: express-UI
+    ports:
+      - 8080:80
+    depends_on:
+      rest:
+        condition: service_healthy
+    environment:
+      - REST_API_URI=http://localhost:8080/api/
+    volumes:
+      - "./frontend/:/etc/nginx/conf.d/"
```

Comment:
Is there any particular erason for mapping the frontend to the container? This doesn't seem to add anything.

Reply:
Yeah that overloads the nginx conf in the ui container to point to the backend appropriately. This was so eventually no ports need to be exposed and we can simulate that the machines run in different docker networks and the ui container can forward request to the backend. For testing that the nginx overload works I decided to leave it this way. The line overloads the current nginx conf with the new one I added in the frontend folder inside test since its relative location based